api/tasks.go

```diff
@@ -474,6 +475,7 @@ type VolumeMount struct {
 	Volume          *string `hcl:"volume,optional"`
 	Destination     *string `hcl:"destination,optional"`
 	ReadOnly        *bool   `mapstructure:"read_only" hcl:"read_only,optional"`
+	Sticky          *bool   `mapstructure:"sticky" hcl:"sticky,optional"`
```

Comment:
I missed this on the original PR, but the `volume_mount` block is at the task level, not at the allocation level where scheduling is controlled. I don't think it has any meaning here. We should remove it from the `structs` package version as well.

Reply:
oof that's a slip on my part, I fix

Reply:
fixed in ac2bf73